project-fortis-interfaces/src/components/Admin/Streams/StreamParamsButtonFormatter.js

```diff
@@ -37,31 +37,42 @@ class StreamParamsButtonFormatter extends React.Component {
     return stream;
   }
 
-  handleShowDetails() {
+  handleShowDetails = () => {
     const stream = this.getStream();
-
     let params = stream.params;
     if (typeof stream.params === 'string') {
       params = JSON.parse(stream.params);
     }
 
+    this.convertParamValuesToString(params);
+
     this.setState({
       isShowDetailOn: true,
       params: params
     });
   }
 
-  getStream() {
+  convertParamValuesToString = (params) => {
+    params.forEach(param => {
+      if (typeof param.value === 'boolean') {
+        param.value = param.value.toString();
+      } else if (param.value.constructor === Array) {
```

Comment:
Another opportunity to make the type checking more explicit using lodash's [isArray](https://lodash.com/docs/4.17.5#isArray).